### PR TITLE
mq, socket close fail log

### PR DESCRIPTION
**Pull request by jeongchanKim**

### CODE REVIEW

File: os/net/lwip/src/api/sockets.c [hunk 1]
```diff
@@ -393,7 +393,12 @@ struct lwip_sock *get_socket(int s, pid_t pid)
 	list = tcb->group->tg_socketlist;
 	sock = (struct lwip_sock *)list.sl_sockets[s].sock;
 
-	if (!sock || !sock->conn) {
+	if (!sock) {
```
Comment (sunghan-chang) [inline os/net/lwip/src/api/sockets.c:396]: Is this enough conditional? Don't you need to compare to `s`?

File: os/net/lwip/src/api/sockets.c [hunk 2]
```diff
@@ -393,7 +393,12 @@ struct lwip_sock *get_socket(int s, pid_t pid)
...
+		LWIP_DEBUGF(SOCKETS_DEBUG, ("get_socket(%d): Socket is not in this thread's group.\n", s + LWIP_SOCKET_OFFSET));
+		set_errno(EBADF);
```
Comment (pillip8282) [inline os/net/lwip/src/api/sockets.c:398]: I think why sock is null would have various reason. So just logging  "sock is null" is enough

File: os/net/lwip/src/api/sockets.c (hunk 1 above)
Comment (jeongchanKim) [reply os/net/lwip/src/api/sockets.c:396]: It tries to get the socket from its task_group based on 's'. So I think it is enough.

File: os/net/lwip/src/api/sockets.c (hunk 2 above)
Comment (jeongchanKim) [reply os/net/lwip/src/api/sockets.c:398]: Modified. Thanks.

**Review by pillip8282 — APPROVED** [review]
lgtm

**Pull request merged**